documentation/faq.asciidoc

```diff
@@ -275,3 +275,27 @@ Caused by: java.lang.IllegalStateException: Cannot read the binlog filename and
 ```
 
 The solution is to indirectly enable the `log_bin` property, activating certain product features in RDS MySQL: read replicas and/or automated backups. Upon activating any of them, the `bin_log` property value will change to `ON` automatically and the connector will be able to complete snapshots successfully.
+
+== Why Debezium PostgreSQL connector causes abnormal consumption of WAL database disk space?
+
+Two reasons that explain the situation
+
+ * Debezium regularly confirms LSN of processed events to the database.
+This is visible as `confirmed_flush_lsn` in `pg_replication_slots` slots table.
+The database is responsible for reclaiming the disk space and the WAL size can be calculated from `restart_lsn` of the same table.
+So if the `confirmed_flush_lsn` is regularly increasing and `restart_lsn` lags then the database does need necessity to reclaim the space.
+Disk space is usually reclaimed in batch blocks.
```

Comment:
I don't quite understand this first item. Will this be resolved automatically when PG reclaims the next batch? I'd expect this not to cause growth in the Gigabyte of ranges, or is it like that? A sentence should be added that this is expected then and no action is needed as PG will handle this periodically by itself.

Reply:
@gunnarmorling The problem is that we've seen reports that the wal grows till GBs before it is reclaimed. I'll add the sentence.